packages/venia-concept/src/reducers/catalog.js

```diff
@@ -9,7 +10,8 @@ export const initialState = {
     rootCategoryId: null,
     currentPage: 1,
     pageSize: 6,
-    prevPageTotal: null
+    chosenFilterOptions: getFilterParams(),
+    appliedFilterOptions: getFilterParams()
```

Comment:
This will get initialized but not updated. Intentional?

Reply:
This will hold filters that are currently being applied on catalog product list. But since the API doesn't yet support it, I can remove it. Chosen filter option keep track of the filters that user edits in the modal and that will be applied to the catalog product list once user clicks "Apply filters".

Reply:
I will remove this for now until we have a proper API support